corelib-utils/src/main/java/eu/europeana/corelib/utils/DateUtils.java

```diff
@@ -53,10 +61,75 @@ public static String format(Date date) {
 	 */
 	public static Date parse(String date) {
 		try {
-			return formatter.parse(date);
+			return DATE_TIME_FORMATTER.parse(date);
 		} catch (ParseException e) {
-			LOG.error(e.getMessage());
+			LOG.warn(e.getMessage());
 		}
 		return null;
 	}
+
+	/**
+	 * Returns true if the string is a year  "yyyy"
+	 *
+	 * @param value value to check
+	 * @return true when the string is a year
+	 */
+	public static boolean isYear(String value) {
+		//check for single year ex: 1930
+		if (StringUtils.isNotEmpty(value)) {
+			try {
+				Year year = Year.of(Integer.parseInt(value));
+				return year != null;
+			} catch (Exception e) {
+				return false;
```

Comment:
It's not a good idea to catch Exception (see also http://sonarqube.eanadev.org:9000/coding_rules?open=squid%3AS2221&rule_key=squid%3AS2221) 
Can we replace it with catch NumberFormatException, or do you think other exceptions can be thrown?

Reply:
It will throw mostly DateTimeParseException. I will catch that one

Reply:
It will throw DateTimeException as well